### PR TITLE
feat(TDP-4884): use ui notifications instead of ours

### PR DESCRIPTION
**Pull request by ncomont**
**Link to the JIRA issue**
https://jira.talendforge.org/browse/TDP-4884

**Please check if the PR fulfills these requirements**
- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] The code coverage on new code is > 75 % for backend and > 95% for frontend
- [ ] The new code does not introduce new technical issues (sonar / eslint)
- [ ] Functional tests have been performed
- [ ] Docker configuration files for config-std or config-cloud profiles are impacted

**Please check the browsers you've tested on**
- [ ] Chrome, Firefox, Safari, Edge, IE11
- [ ] No, that's bad, this PR should not be merged !
- [ ] No, and no need to (backend changes only)

### CODE REVIEW

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/21/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/21/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/24/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/24/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/36/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/36/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/57/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/57/)

File: dataprep-webapp/src/vendor.js [hunk 1]
```diff
@@ -67,10 +67,6 @@ import 'ng-file-upload/dist/angular-file-upload.min';
 import 'ng-sortable/dist/ng-sortable.min.css';
 import 'ng-sortable';
 
-// https://github.com/jirikavi/AngularJS-Toaster
-import 'angularjs-toaster';
-import 'angularjs-toaster/toaster.scss';
```
Comment (charlesnguyen) [inline dataprep-webapp/src/vendor.js]: data-prep\dataprep-webapp\src\app\css\components\_toaster.scss has to be removed

File: dataprep-webapp/src/app/services/state/message/message-state-service.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ 9 rue Pages 92150 Suresnes, France
+
+ ============================================================================*/
+
+
+export const messageState = {
+	messages: [],
+};
+
+export class MessageStateService {
```
Comment (charlesnguyen) [inline dataprep-webapp/src/app/services/state/message/message-state-service.js]: Coul you add unit tests for this service?

File: dataprep-webapp/src/app/components/widgets-containers/notifications/notifications-controller.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+ This source code is available under agreement available at
+ https://github.com/Talend/data-prep/blob/master/LICENSE
+
+ You should have received a copy of the agreement
+ along with this program; if not, write to Talend SA
+ 9 rue Pages 92150 Suresnes, France
+
+ ============================================================================*/
+
+export default class NotificationsCtrl {
```
Comment (charlesnguyen) [inline dataprep-webapp/src/app/components/widgets-containers/notifications/notifications-controller.js]: Could you add unit test for notifications component please?

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/59/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/59/)

**Pull request merged**